src/loaders/gtdb/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,223 @@
+"""
+PROTOTYPE
+
+This script involves processing computed genome attributes and organizing them into a structured format suitable
+for importing into ArangoDB. The resulting JSON file will be used to upsert the database with the parsed genome
+attribute data.
+
+Note: The database was previously created using a JSON file generated by the gtdb_genome_attribs_loader.py script.
+      It is important to ensure that the arguments "--load_ver" and "--kbase_collection" match the ones used in the
+      gtdb_genome_attribs_loader.py script in order to generate the same key for the corresponding Arango document.
```

Comment:
Or is this specific to GTDB? It seems like it could work for any collection.

Reply:
I think we should follow the naming protocol (`/kbase_collectoin/load_ver`) for any data source. Even with on load version, we should give the version a name (something like `version_1`). So that paths are consistent across different data sources.

Reply:
But why do then need to be the name as the input to the `gtdb_genome_attribs_loader.py` script?

Reply:
The doc key from metadata is formed as `md5_string(f"{kbase_collection}_{load_version}_{genome_id}")`. I want computed attributes docs to have the same key as metadata docs so we can upsert.

Reply:
Covered in comment below this review

Reply:
Subsumed in subsequent review